modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -94,19 +111,276 @@ definition {
 	}
 
 	@priority = 4
-	test PicklistScopeHasCorrectOptions {
+	test CanCreateEndpointWithSuccessAlert {
 		property portal.acceptance = "true";
 
-		task ("Given the API Application is created") {
-			ApplicationAPI.createAPIApplication(
-				baseURL = "test3",
-				title = "test3");
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
 		}
 
+		task ("When I enter a path and Scope to create the API endpoint") {
```

Comment:
In the orignal GWT, we somtimes create the endpoint using Add an API Endpoint button, and sometimes with the PLUS button, please help update accordingly, thanks